step.sh

```diff
@@ -168,6 +199,46 @@ function device_farm_run {
     local run_response=$(aws devicefarm schedule-run "${run_params[@]}")
     echo_info "Run started for $run_platform!"
     echo_details "Run response: '${run_response}'"
+
+    # Obtain the ARN for the run from the schedule-run request
+		local run_arn=$(echo "${run_response}" | jq -r .run.arn)
```

Comment:
Saw your comment about whitespace. The code I wrote in this file is indented with 4 spaces per indent, so you just need to replace the tabs with 4 spaces to keep it consistent. (Now that I'm looking at it I noticed that some of the code I forked from bitrise uses tabs too so I will clean that up later.)